src/components/Episodes/Episodes.jsx

```diff
@@ -12,16 +13,18 @@ export default function Episodes(){
 
 	return (
 		<article>
-			<ol>
-				{ episodes }
-			</ol>
-			<nav>
-				<button aria-label="Previous episode." />
-				<button aria-label="Next episode." />
+			<EpisodesProvider>
 				<ol>
```

Comment:
let's whack similar `aria` accessibility stuff on this to what we've done on the `<Characters>` carousel

Reply:
OK i've added the same accessibility attributes as we have in Character, all working fine except aria-hidden and I am very confused as to why there is an issue.

Using this gives us the correct result (episode 0 is active and not hidden, episode 1 is hidden):
- const flippedActive = !isActive;
- aria-hidden={ flippedActive.toString() }

Using aria-hidden={ !isActive.toString() } has them both set to false.

I just inspected our character list items and they're having the same problem. 
Any idea why the difference? I imagine there's some timing thing with doing the calculation (!) in the render return....hmmm